### DISCUSSION TITLE
opgg ad removal  feature

### DISCUSSION THREAD

**Discussion by potentate** [MidKnightXI/opgg-ads-remover#33] [Ideas]
maybe try looking into if theres a way to keep their built in "3 hour ad removal" forever instead of just for 3 hours

**Comment by MidKnightXI** [comment]
Yeah I just saw that earlier, but I'm on a good way to block every ads they can put and maybe "forever" I hope :D

**Reply by MidKnightXI** [reply]
I thought about making a local proxy server if the thing I'm currently trying is not working

**Reply by MidKnightXI** [reply]
Also their "ad free" stuff needs an account for it, I guess there would be a way to maybe bypass that as it is surely a boolean that tells the frontend to display ads or not when they are fetching for user infos

**Reply by MidKnightXI** [reply]
But I think it is not a durable patch imo
